### PR TITLE
Feature/graph context

### PR DESCRIPTION
**Pull request by hiroshinoji**
Support a command './waf graph' with GraphContext.

### CODE REVIEW

**Comment by hiroshinoji** [comment]
See #43

**Comment by hiroshinoji** [comment]
Currently, there is a problem with an old-style wscript that defines tasks in "experiment" instead of "build", because GraphContext is a subclass of ExperimentContext.

File: maflib/core.py [hunk 1]
```diff
@@ -292,6 +294,201 @@ def _not_deleted_any_files_in(n):
...
+        # dot.write("size=\"50,50\";\n")
+        dot.write(meta_nodes.render_graphviz(node_indexer, self) + "\n")
+        dot.write(meta_tasks.render_graphviz() + "\n")
+        for link in links:
+            dot.write("%s->%s[arrowsize=0.3]\n" % (link[0], link[1]))
+        dot.write("}")
+
+        dot.seek(0)
+
+        import subprocess
```
Comment (beam2d) [inline maflib/core.py]: `import` should be placed at the head of the source file.

**Comment by beam2d** [comment]
Great job! Thank you!

**Pull request merged**